### ISSUE TITLE
Express-Gateway should fail to boot if a policy is missing an action

### ISSUE THREAD

**Issue by XVincentX** [ExpressGateway/express-gateway#415]
The following (wrong) configuration file for the gateway is making eg boot correctly, although the defined policy is missing an action (which should be mandatory)

```yaml
http:
  port: 8080
admin:
  port: 9876
  hostname: localhost
apiEndpoints:
  api:
    host: '*'
serviceEndpoints:
  httpbin:
    url: 'https://httpbin.org'
policies:
  - proxy
  - rewrite
pipelines:
  - name: basic
    apiEndpoints:
      - api
    policies:
      - rewrite:
          - condition:
              name: 'match'
              match: '/calendar/:test/hello'
          - action:
              rewrite: 'hello/:test/calendar'

      - proxy:
          - action:
              serviceEndpoint: httpbin
              changeOrigin: true
```

**Comment by XVincentX** [comment]
This could be done leveraging the JSON schema.

**Comment by XVincentX** [comment]
This will automatically be closed once we make #561 happen.
